docs/source/system_replication/system_replication.rst

```diff
@@ -0,0 +1,114 @@
+and copy the contents to the USB drive.
+
+Getting Image
+============
+
+To take system image of an x64 NI Linux RT system, plug the USB drive and change boot
+order to boot into it.
+
+When the following boot menu appears, select **Get Image** option.
+
+   .. image:: media/boot_menu.png
+
+Enter a name for the image (make sure there are no spaces or special characters
+except `-`) or accept the provided name.
+
+   .. image:: media/get_image_name.png
+
+Getting the image can take a few minutes depending on the amount of user data and
+write speed of USB drive. The progress bar will keep moving to show progress.
+
+   .. image:: media/get_image_progress.png
+
+The image will be saved in the USB drive under `Images/<NameOfSystemImage>`.
+
+The tool will estimate if the available space on USB drive will be sufficient to
+store the contents of all partitions and if not, will error out.
+
+Once it's done, the following prompt will be shown. The USB drive can then be removed
+and the device rebooted.
+
+   .. image:: media/get_image_done.png
+
+Setting Image
+============
+
+To apply this system image on another NI Linux RT system of the same model, plug
+the USB drive and boot into it, then select **Set Image** menu option.
+
+All compatible images would be listed; select one by entering the corresponding
+number and press Enter.
+
+   .. image:: media/set_image_name.png
+
+Confirm that target can be reimaged by entering `y`.
+
+   .. image:: media/set_image_confirmation.png
+
+A progress bar will keep moving to show progress.
+
+   .. image:: media/set_image_progress.png
+
+Once it's done, the following prompt will be shown. The USB drive can then be removed
+and the device rebooted.
+
+   .. image:: media/set_image_done.png
+
+Using Answers File
+==================
+
+A file named `ni_provisioning.answers` can be placed in the USB drive to disable
+interactive user prompts. Each prompt has a corresponding variable that can be
+set to desired value.
+
+E.g., following `ni_provisioning.answers` when placed in root of USB drive will
+disable all user prompts in `Get Image` or `Set Image`.
+
+When "Get Image" is selected in boot menu, the saved image will be named
+"NameOfSystemImage" and target will be powered off.
+
+When "Set Image" is selected in boot menu, an image on the USB drive named
+"NameOfSystemImage" will be applied and target will be powered off.
+
+.. code:: text
+
+   #NI_PROVISIONING_ANSWERS_V1
+
+   # Setting this to "y" would disable prompt asking for confirmation to repartition. Not applicable in "Get Image"
+   PROVISION_REPARTITION_TARGET="y"
+   # System Image name used for "Get Image" and "Set Image". No spaces, no special characters except '-'
+   PROVISION_SYSTEMIMAGE_NAME="NameOfSystemImage"
+   # Controls what action is performed after "Get Image" or "Set Image" is done. Accepted values are "reboot", "poweroff", and "shell".
```

Comment:
Controls what action is performed after **the** "Get Image" or "Set Image" **operation** is **complete**...